### PR TITLE
Document resolution process

### PR DESCRIPTION
**Pull request by tconway**
This might be better suited somewhere else...

### CODE REVIEW

File: documents/markdown/resolution_process.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+From the organisation information, it will determine whether there is a Redirect Id Type provided as part of the reference links.
+
+If a Redirect Id Type is provided, then the resolution service will **Redirect**
+
+If a Redirect Id Type is not provided (or the asset does not have a Source Id Type that matches the provided Redirect Id Type), 
+then the resolution service will **Render**
+
+### Redirect
+The Redirect Id Type is used to look up the **reference link** to redirect to. 
+The reference link must be a valid url, and allows for optional formatting of the source id within the url structure.
```
Comment (Jimflip) [inline documents/markdown/resolution_process.md]: Not sure 'formatting' is the right word for this, although I appreciate the python function used is called that.

File: documents/markdown/resolution_process.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# The Open Permissions Platform Resolution Workflow
```
Comment (Jimflip) [inline documents/markdown/resolution_process.md]: So a diagram (flow chart/ activity diagram) would help. Also rename this as LLD I think.

**Comment by tconway** [comment]
Renamed to LLD and added diagrams for S0 and S1 hubkeys.

File: documents/markdown/low_level_design.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+An Open Permissions Platform hub key can be resolved in one of three ways
+
+1. Redirect to a page displaying specific asset information
+2. Redirect to a page displaying generic organisation information
+2. Render information about the asset and its organisation
+
+The resolution process used is determined by the Reference Links set up for an organisation.
+
+## UML
```
Comment (Jimflip) [inline documents/markdown/low_level_design.md]: UML?

**Comment by tconway** [comment]
Is this okay to be merged?

**Pull request merged**